importers/imports.go

```diff
@@ -285,6 +285,27 @@ func NewDefaultImports() Collection {
 	return col
 }
 
+// NewNullableEnumImports returns imports collection for nullable enum types.
+func NewNullableEnumImports() Collection {
```

Comment:
Is there a reason the name of this function starts with **New**? Perhaps we can just call it `NullableEnumImports`

Reply:
No reason, just an old habit from pre-Go days. Fixed.